tools/cache_creator/cache_creator.h

```diff
@@ -24,8 +24,91 @@
  **********************************************************************************************************************/
 #pragma once
 
+#include "util/palMetroHash.h"
+#include "include/binary_cache_serialization.h"
+
+// These Xlib defines conflict with LLVM.
+#if defined(Bool)
+#undef Bool
```

Comment:
I think you can say #undef without needing to wrap it in an #ifdef, right?

Reply:
Thanks, I missed this